frontend/src/routes/Chats.tsx

```diff
@@ -148,10 +151,44 @@ export default function Chats() {
                             name="users[]"
                             value={currentUser.id}
                           />
+
+                          <Hr className="my-3"></Hr>
+
+                          <input
+                            type="checkbox"
+                            name="hasPassword"
+                            id="hasPassword"
+                            className="mr-1 mb-3"
+                            onChange={() => setHasPassword(!hasPassword)}
+                          />
+                          <Typography variant="sm" as="label">
+                            Protect with password
+                          </Typography>
```

Comment:
Uma pequena sugestaao, veja oq acha de aalinharmos a esquerda:
```suggestion
                          <div className="text-left">
                            <input
                              type="checkbox"
                              name="hasPassword"
                              id="hasPassword"
                              className="mb-3 mr-1"
                              onChange={() => setHasPassword(!hasPassword)}
                            />
                            <Typography variant="sm" as="label">
                              Protect with password
                            </Typography>
                          </div>
```